src/viewer.js

```diff
@@ -1680,6 +1699,18 @@ $.extend( $.Viewer.prototype, $.EventSource.prototype, $.ControlDock.prototype,
         return this;
     },
 
+    /**
+     * Force the viewer to reset it's size to it's container.
```

Comment:
This is just a work-in-progress, so maybe it's silly to be noting spelling errors, but just in case this comment survives, they should both be `its`.